views/budget_detail.pug

```diff
@@ -0,0 +1,15 @@
+extends layout
+
+block content
+
+  h1 Budget: #{budget.name}
+  p Income: #{budget.income}
+  p Expeneses: #{budget.expenses}
+  - var total = budget.income - budget.expenses
```

Comment:
This could be a virtual property on Budget.

Reply:
Looks okay to me.